test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -249,14 +249,20 @@ def _run_dependency_check_test(image, ec2_connection, processor):
     short_fw_version = re.search(r"(\d+\.\d+)", image).group(1)
 
     allow_openssl_cve_fw_versions = {
-        "tensorflow": ["1.15", "2.3", "2.4", "2.5", "2.6"],
-        "mxnet": ["1.9"],
-        "pytorch": [],
-        "huggingface_pytorch": ["1.8"],
-        "huggingface_tensorflow": ["2.4"]
+        "tensorflow": {
+            "1.15": ["cpu", "gpu", "neuron", "eia"],
```

Comment:
processor seems to come from the test directly. Moreover, in this script, it was just used to name the container it seems. I don't believe that we have a dependency check specifically for `eia` - you may need to update `test_dependency_check_cpu` if you need to handle EI separately.